internal/config/config.go

```diff
@@ -0,0 +1,47 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/joho/godotenv"
+)
+
+type AppConfig struct {
+	MySQLHost     string
```

Comment:
Can we have a better naming for these ?
```suggestion
	DBHost     string
```

Reply:
Nice one , Sure

Reply:
Done